data/human/events.txt

```diff
@@ -3061,3 +3061,138 @@ mission "Martini Recovers"
 	on offer
 		event "martini recovers" 45
 		fail
+
+
+
+mission "Hunted Warning"
```

Comment:
why did you move this? doesnt feel like it should be done in this PR

Reply:
I'm fine with moving it from `jobs` to `events`, but I would prefer it be done in a separate PR rather than having one commit that adds the Hai thing and also moves two missions from `human/jobs` to `human/events`, with the only relation being that those missions inspired the new Hai mission.

Reply:
Aight, I'll move that into it's own PR in a bit.